### DISCUSSION TITLE
payloadjs provider to support the nextjs commerce

### DISCUSSION THREAD

**Discussion by megetron** [payloadcms/payload#2451] [Feature Requests & Ideas]
have to say that i love the way you design the payloadjs headless CMS, where it really something that integrates well to existing apps that uses express and mongo as their main tech stack.

I develop some kind of marketing website that compares products.
considering using headless CMS with nextjs commerce frontend (to allow static pages).
(https://nextjs.org/commerce)

to make this happend i would need to write a payloadjs provider for the nextjs commerce project.

i wonder if it already exists if it can be registered as a feature request?
thanks,

**Comment by jacobsfletch** [comment]
@megetron love to hear it! We definitely will want to wire up a Payload provider for the Next.js commerce project. I just marked this is a roadmap item so we don't lose track. We _do_ already have a complete end-to-end e-commerce solution of our own using Payload, Next.js, and Stripe—which will be our go-to resource regardless. Check it out! https://github.com/payloadcms/template-ecommerce

We also have a growing number of Next.js integrations listed in our "examples" directory here https://github.com/payloadcms/payload/tree/master/examples
